src/dispatch/project/models.py

```diff
@@ -65,10 +76,12 @@ class ProjectCreate(ProjectBase):
 
 class ProjectUpdate(ProjectBase):
     send_daily_reports: Optional[bool] = Field(True, nullable=True)
+    restrict_stable_to_id: Optional[int]
```

Comment:
Why not just pass the whole object back? e.g. `IncidentPriorityRead` that's the pattern we've typically followed before.

Reply:
Must be related to the quirkiness of the way I overcame the circular logic, but this was the only way I could get the UI to update that attribute.